### PR TITLE
Like feature done :) 

### PR DESCRIPTION
**Pull request by joordy**
Made like feature, you get match.ejs when like from both sides, otherwise nothing.

- Like from you + like from other => match
- Like from you + no like from other = adds in your liked. When both are in liked array, => match
- No like from you => no match

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -343,48 +341,76 @@ input:focus {
...
+  width: 8em;
+  height: 11em;
+  object-fit: cover;
+  left: 4em;
+  transform: rotate(-25deg);
+  /* width: 40%;
+  max-width: 10em;
+  height: auto;
+  border-radius: 5px;
+  position: absolute;
```
Comment (veerleprins) [inline static/css/styles.css:409]: This is out commented, can't it be taken away then?

Comment (joordy) [reply static/css/styles.css:409]: Yes! Forgot to clean that up. I will fix it in the next pull request I make!

Comment (veerleprins) [reply static/css/styles.css:409]: Sounds good! 😄

**Comment by veerleprins** [comment]
Good to see that you have adjusted your code so that it should now be liked by both users!

**Comment by joordy** [comment]
> are you still going to use this, or did you also remove these at the same time with the comment @veerleprins made?
> 
> good to know it works, we are almost there!

How do you mean? The CSS Code? I will not be using that again, so it will be removed soon :)

**Pull request merged**